### ISSUE TITLE
error when linting

### ISSUE THREAD

**Issue by xZarisx** [AtomLinter/linter-eslint#231]
```
/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:47 activate linter-eslint
/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:266 [Linter-ESlint] error loading global eslint
/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:267 Error: Command failed: npm config get prefix
/bin/sh: npm: command not found

/bin/sh: npm: command not found

  at checkExecSyncError (child_process.js:445:13)
  at execSync (child_process.js:485:13)
  at Object.module.exports.findGlobalNPMdir (/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:226:26)
  at Object.module.exports.activate (/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:37:63)
  at Package.module.exports.Package.activateNow (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:245:19)
  at /Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:226:30
  at Package.module.exports.Package.measure (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:169:15)
  at Package.module.exports.Package.activate (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:218:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/package-manager.js:486:21)
  at /Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/package-manager.js:469:29

index.js:46 Window load time: 1546ms
/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:215 [Linter-ESLint] local `eslint` not found
/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:216 Error: Could not find `eslint` locally installed in /Users/mdpoulson/Projects/Sandboxes/deleteMe2 or any parent directories
  at Object.module.exports.requireLocalESLint (/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:217:15)
  at Object.module.exports.requireESLint (/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:181:17)
  at Object.provider.lint (/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:80:32)
  at Promise.then._this.emitter.emit.linter (/Users/mdpoulson/.atom/packages/linter/lib/linter-registry.coffee:59:22)
  at LinterRegistry.triggerLinter (/Users/mdpoulson/.atom/packages/linter/lib/linter-registry.coffee:58:16)
  at /Users/mdpoulson/.atom/packages/linter/lib/linter-registry.coffee:52:17
  at Array.map (native)
  at /Users/mdpoulson/.atom/packages/linter/lib/linter-registry.coffee:50:27

/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:142 [Linter-ESLint] error while linting file
/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:143 Cannot read property 'match' of undefined
/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:144 TypeError: Cannot read property 'match' of undefined
    at TokenizedBuffer.module.exports.TokenizedBuffer.indentLevelForLine (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:680:23)
    at DisplayBuffer.module.exports.DisplayBuffer.indentLevelForLine (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/display-buffer.js:817:35)
    at TextEditor.module.exports.TextEditor.indentLevelForLine (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2284:33)
    at TextEditor.module.exports.TextEditor.indentationForBufferRow (/Users/mdpoulson/Downloads/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2256:19)
    at /Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:112:44
    at Array.map (native)
    at /Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:110:18
    at exports.allowUnsafeNewFunction (/Users/mdpoulson/.atom/packages/linter-eslint/node_modules/loophole/lib/loophole.js:25:14)
    at Object.provider.lint (/Users/mdpoulson/.atom/packages/linter-eslint/lib/linter-eslint.coffee:107:13)
    at Promise.then._this.emitter.emit.linter (/Users/mdpoulson/.atom/packages/linter/lib/linter-registry.coffee:59:22)
```

**Comment by Arcanemagus** [comment]
Edited to place in code block so it's readable.

It _looks_ like it can't find an `eslint` at all, but I'm not really sure. Do you have `npm` available on your system?

**Comment by xZarisx** [comment]
Yes I do

On Thursday, October 15, 2015, Landon Abney <notifications@github.com>
wrote:

> Edited to place in code block so it's readable.
>
> It *looks* like it can't find an eslint at all, but I'm not really sure.
> Do you have npm available on your system?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AtomLinter/linter-eslint/issues/231#issuecomment-148458695>
> .
>


-- 
-Matthew D. Poulson
xzarisx@gmail.com
pou07002@byui.edu
(208) 403-8770

**Comment by Arcanemagus** [comment]
Sorry for not getting back to you earlier, can you update to the latest version? It has been updated to use `eslint` >= v1.x.x.

**Comment by steelbrain** [comment]
Fixed by v5.0.0
